_includes/partners/header.html

```diff
@@ -5,20 +5,16 @@
         class="usa-nav__link"
         href="/"
         title="Login.gov"
-        aria-label="Login.gov"
-        tabindex="0"
       >
-        <span>Login.gov</span>
+        Login.gov
       </a>
       <span class="links-separator">&#124;</span>
       <a
         class="usa-nav__link"
```

Comment:
Per https://github.com/18F/identity-site/pull/826#discussion_r825897326, should add `usa-link` class here as well, for correct link coloring:

```suggestion
        class="usa-nav__link usa-link"
```

Reply:
ok fixed